src/PDE/CompFlowProblem/RayleighTaylor.h

```diff
@@ -0,0 +1,521 @@
+// *****************************************************************************
+/*!
+  \file      src/PDE/CompFlowProblem/RayleighTaylor.h
+  \author    F. Gonzalez
```

Comment:
Aren't we removing these to support a more traditional open source development model?